test/index.spec.js

```diff
@@ -460,6 +460,16 @@ describe('helpers', () => {
         port: '1234'
       })
     })
+
+    it('returns a node friendly address with dns', () => {
+      expect(
+        multiaddr('dns4/wss0.bootstrap.libp2p.io/tcp/443').nodeAddress()
```

Comment:
This is not a valid multiaddr, missing the initial `/`

Reply:
Not sure how this test can be passing currently? Seems some checks are missing, at least this test should be failing.

Reply:
I will add the `/`.

Anyway.  it is possible to do the following:

```js
const ipfs = ipfsAPI('ip4/127.0.0.1/tcp/5002')
```

Even, with the code in master 😕

Reply:
Let's add a test and checks that it starts with a `/` as well then